painter/src/root_painter.py

```diff
@@ -360,7 +360,7 @@ def get_seg_path(self, fname=None):
     def get_train_seg_path(self, fname=None):
         if fname is None:
             fname = self.fname
-        seg_fname = fname.replace('.nrrd', '.nii.gz')
+        seg_fname = fname.replace('.nrrd', '.nii.gz')  # @TODO: How to handle this with .nii files?
```

Comment:
.nii is only something we need to worry about for the input dataset. As segmentations are generated  by RootPainter3D we can rely on them being .nii.gz.

Reply:
OK, but then I can remove the TODOs I created? There are not any other stuff that need changing?

Reply:
Actually the fname might end in .nrrd .nii.gz or .nii as this comes from the original dataset.
We want to get it to .nii.gz every time.

So to make this correct I believe it should be changed to: 

seg_fname = fname.replace('.nrrd', '.nii.gz').replace('.nii', '.nii.gz')

Reply:
Wait... that is also wrong. haha

Reply:
Perhaps we should write a function to handle this. i.e

from file_utils import set_ext
seg_fname = set_ext(fname, '.nii.gz')

Because its actually a bit tricky doing this with multiple replacements.

Reply:
Yeah, I figured it was a little bit more tricky than the other ones, as we are now having three possible input formats instead of two :P

Reply:
Yeah, and maybe we will add more file extensions in the future so perhaps a function is the way to go.